docs/dev/setup.md

```diff
@@ -4,8 +4,8 @@
 
 The development workflow is streamlined for Linux thanks to the `conda/conda` Docker images used in the upstream CI.
 
-1. Clone `conda/conda` and `conda/conda-libmamba-solver` to your preferred locations
-   (e.g. `~/devel/conda` and `~/devel/conda-libmamba-solver`, respectively).
+1. Clone `conda/conda`, `mamba/mamba`, `conda/conda-libmamba-solver` to your preferred locations
```

Comment:
```suggestion
1. Clone `conda/conda`, `mamba-org/mamba`, `conda/conda-libmamba-solver` to your preferred locations
```

Can we write the instructions in a way that the mamba dev stuff is an alternative workflow?

Reply:
Change made.